micro_ros_setup/scripts/create_firmware_ws.sh

```diff
@@ -84,7 +85,11 @@ fi
 
 # build the dev_ws
 . $(dirname $0)/clean_env.sh
-pushd $FW_TARGETDIR/$DEV_WS_DIR >/dev/null
+if [ $RTOS != "host" ]; then
```

Comment:
I think this is wrong. For the host, the dev_ws build should just be empty, so building does nothing.

If you instead build the entire firmware folder, that totally changes what create_firmware_ws.sh does.

Reply:
True, solved in the last commits.